CHANGELOG.md

```diff
@@ -26,9 +26,14 @@
 - `jmxreceiver`: Remove properties & groovyscript parameters from JMX Receiver. Add ResourceAttributes & LogLevel parameter to supply some of the removed functionality with reduced attack surface (#9685)
 - `resourcedetectionprocessor`: 'gke' and 'gce' resource detectors are replaced with a single 'gcp' detector (#10347)
 - `pkg/stanza`: Removed reference to deprecated `ClusterName` (#10426)
+- `tailsamplingprocessor`: New sampler added that allows to sample based on minimum number of spans
```

Comment:
This should go line 17 or so (it's an enhancement and also one for the unreleased version)